contracts/upgradeable-polymesh-ink/src/lib.rs

```diff
@@ -626,6 +626,11 @@ upgradable_api! {
                     .ok_or(PolymeshError::MissingIdentity)
             }
 
+            /// Returns the next [`AssetId`] for the caller.
+            pub fn get_our_asset_id() -> PolymeshResult<AssetId> {
+                Self::get_asset_id(ink::env::account_id::<PolymeshEnvironment>())
+            }
+
```

Comment:
I think we should be able to return the asset id from the `create_asset_**` method.

Reply:
Also we should most like add `_next_` to the name of these methods to make it clear that it will return the asset id of the next created asset.